### PR TITLE
Made port selectable

### PR DESCRIPTION
**Pull request by uweseimet**
This PR resolves https://github.com/akuker/RASCSI/issues/143. It introduces a -p option for rascsi and rasctl. Default is 6868, just like before.

### CODE REVIEW

File: src/raspberrypi/rasctl.cpp [hunk 1]
```diff
@@ -238,6 +240,14 @@ int main(int argc, char* argv[])
 				hostname = optarg;
 				break;
 
+			case 'p':
+				port = atoi(optarg);
+				if (!port || port > 65535) {
```
Comment (akuker) [inline src/raspberrypi/rasctl.cpp:245]: What if the port is negative?

File: src/raspberrypi/rascsi.cpp [hunk 2]
```diff
@@ -759,6 +759,14 @@ bool ParseArgument(int argc, char* argv[])
 				continue;
 			}
 
+			case 'p':
+				port = atoi(optarg);
+				if (!port || port > 65535) {
```
Comment (akuker) [inline src/raspberrypi/rascsi.cpp:764]: What if the port is negative?

**Comment by akuker** [comment]
FYI: Re-ran robot tests - works as expected.

File: src/raspberrypi/rasctl.cpp (hunk 1 above)
Comment (uweseimet) [reply src/raspberrypi/rasctl.cpp]: Just fixed this (both locations)

**Review by akuker — APPROVED** [review]

**Pull request merged**